### PR TITLE
175-fix

### PR DESCRIPTION
**Pull request by Drulac**

### CODE REVIEW

File: src/backend/runner-base.js [hunk 1]
```diff
@@ -94,16 +94,23 @@ module.exports = class BaseRunner {
...
 	 * @desc Get and returns the Synchronous executor, of a class and kernel
 	 * Which returns the result directly after passing the arguments.
-	 * 
+	 *
 	 */
 	buildKernel(fn, settings) {
 		settings = Object.assign({}, settings || {});
-		const fnString = fn.toString();
+
+		var fnString = '';
```
Comment (robertleeplummerjr) [inline src/backend/runner-base.js]: `const fnString = fn.toString();` should be sufficient on a string or a function.

File: src/core/gpu.js [hunk 2]
```diff
@@ -83,7 +83,7 @@ class GPU extends GPUCore {
 		if (typeof fn === 'undefined') {
 			throw 'Missing fn parameter';
 		}
-		if (!utils.isFunction(fn)) {
+		if (!utils.isFunction(fn) && typeof fn !== 'string') {
```
Comment (robertleeplummerjr) [inline src/core/gpu.js]: The logic that checks for `isFunction` should probably contain the logic for string or not.  Can you more that there?

File: src/backend/runner-base.js (hunk 1 above)
Comment (Drulac) [reply src/backend/runner-base.js]: I will do that

File: src/core/gpu.js (hunk 2 above)
Comment (Drulac) [reply src/core/gpu.js]: No, if fn is a string, that will fire the if and throw an error

**Pull request merged**